src/x/__tests__/xHilbertTransform.test.ts

```diff
@@ -1,34 +1,32 @@
 import { xHilbertTransform } from '../../index';
 
-describe('test hilbert transform of cos(t)', () => {
-  const length = 50;
+describe('test hilbert transform', () => {
```

Comment:
Keep the old test case as well. If the old testcase was correct and there is no way of have the FFT approach with non power of 2 there should be a new method xFastHilbertTransform that could only be applied on power of 2 data size.